plugin/aws/s3Bucket.go

```diff
@@ -45,12 +114,14 @@ func listObjects(ctx context.Context, client *s3Client.S3, bucket string, prefix
 		numObjects,
 	)
 
+	// resp.CommonPrefixes represents all of the object keys
 	for _, p := range resp.CommonPrefixes {
 		commonPrefix := awsSDK.StringValue(p.Prefix)
 		name := strings.TrimPrefix(commonPrefix, prefix)
 		if name != "/" {
 			name = strings.TrimSuffix(name, "/")
```

Comment:
This will probably do bad things for your example `//`.

Reply:
`//` => `#/#` in Wash. That's b/c the prefix there will be `/`, so name will == `/`.